mage_ai/frontend/oracle/icons/index.tsx

```diff
@@ -1514,6 +1514,18 @@ export const Union = build([{
   viewBox: '0 0 18 20',
 });
 
+// https://www.flaticon.com/free-icons/file-upload
+export const Upload = build([{
```

Comment:
Please replace the Upload icon with this:
```
build([{
  d: 'M9.46967 1.96967C9.76256 1.67678 10.2374 1.67678 10.5303 1.96967L14.0303 5.46967C14.3232 5.76256 14.3232 6.23744 14.0303 6.53033C13.7374 6.82322 13.2626 6.82322 12.9697 6.53033L10.75 4.31066V13.5C10.75 13.9142 10.4142 14.25 10 14.25C9.58579 14.25 9.25 13.9142 9.25 13.5V4.31066L7.03033 6.53033C6.73744 6.82322 6.26256 6.82322 5.96967 6.53033C5.67678 6.23744 5.67678 5.76256 5.96967 5.46967L9.46967 1.96967Z',
  clipRule: 'evenodd',
  fillRule: 'evenodd',
}, {
  d: 'M3 12.25C3.41421 12.25 3.75 12.5858 3.75 13V16C3.75 16.6904 4.30964 17.25 5 17.25H15C15.6904 17.25 16.25 16.6904 16.25 16V13C16.25 12.5858 16.5858 12.25 17 12.25C17.4142 12.25 17.75 12.5858 17.75 13V16C17.75 17.5188 16.5188 18.75 15 18.75H5C3.48122 18.75 2.25 17.5188 2.25 16V13C2.25 12.5858 2.58579 12.25 3 12.25Z',
  clipRule: 'evenodd',
  fillRule: 'evenodd',
}], {
  fill: 'none',
  viewBox: '0 0 20 20',
});
```

Reply:
Please also add `Upload` to the `ICONS` constant in the `Icons.stories.tsx` file. You'll see many other icons we added there. It's so we can visually see what icons are available in Storybook.